schemas/test/en/test_supplementary_data_non_list_items.json

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
It would be good to demo https://github.com/ONSdigital/eq-questionnaire-runner/pull/1122 this using supplementary data. Either in the file that was introduced there or this one.

Reply:
Removed the guidance from one of the datasets so can be tested with and without 👍